iac/stacks/common.py

```diff
@@ -21,6 +21,36 @@
 
 
 class CommonStack(Stack):
+    def _database(self, id: str, version: rds.PostgresEngineVersion) -> rds.DatabaseInstance:
```

Comment:
I don't see us adding more than 2 databases, so no need for a function. To keep it DRY, you can put common attributes in a `dict` and deconstruct it (`**common_attrs`) in the `rds.DatabaseInstance` constructor call.

Reply:
Started down this road but the part of the function that generates the secret has to be repeated, and that's more than half the function; thinking it's not worth a refactor?

Reply:
That is fine.